### PR TITLE
Small CMake cleanups

### PR DESCRIPTION
**Pull request by Orphis**

### CODE REVIEW

File: Externals/portaudio/CMakeLists.txt [hunk 1]
```diff
@@ -316,8 +316,9 @@ ELSE()
 
   ENDIF()
 
-  SET(PA_PKGCONFIG_LDFLAGS "${PA_PKGCONFIG_LDFLAGS} -lm -lpthread")
-  SET(PA_LIBRARY_DEPENDENCIES ${PA_LIBRARY_DEPENDENCIES} m pthread)
+  find_package(Threads)
```
Comment (ligfx) [inline Externals/portaudio/CMakeLists.txt]: Why the change?

File: Source/Core/AudioCommon/CMakeLists.txt [hunk 2]
```diff
@@ -6,9 +6,11 @@ set(SRCS	AudioCommon.cpp
 
 set(LIBS "")
 
+find_package(OpenSLES)
 if(OPENSLES_FOUND)
+	message(STATUS "OpenSLES found, enabling OpenSLES sound backend")
 	set(SRCS ${SRCS} OpenSLESStream.cpp)
-	set(LIBS ${LIBS} ${OPENSLES_LIBRARIES})
+	set(LIBS ${LIBS} OpenSLES::OpenSLES)
```
Comment (ligfx) [inline Source/Core/AudioCommon/CMakeLists.txt]: I like this. Are you going to move the rest of the audio backends here too?

Comment (Orphis) [inline Source/Core/AudioCommon/CMakeLists.txt]: It's a possibility!

File: Externals/portaudio/CMakeLists.txt (hunk 1 above)
Comment (Orphis) [inline Externals/portaudio/CMakeLists.txt]: Building Android headless client went through there and didn't like the -lpthread flag.

Comment (ligfx) [inline Externals/portaudio/CMakeLists.txt]: jeez
